reflex/components/forms/input.py

```diff
@@ -50,25 +51,28 @@ class Input(ChakraComponent):
     # "lg" | "md" | "sm" | "xs"
     size: Var[str]
 
+    # Called when the input value changes.
+    on_change: EventHandler[EVENT_ARG.target.value]
```

Comment:
this is a cool idea. i think it'd be neat to extend this syntax to optionally support custom signature mapping like requested in #1762

Reply:
Will look into that